addons/survey/static/src/js/survey_session_manage.js

```diff
@@ -238,7 +238,7 @@ publicWidget.registry.SurveySessionManage = publicWidget.Widget.extend(SurveyPre
             if ($(ev.currentTarget).data('showResults')) {
                 document.location = `/survey/results/${encodeURIComponent(self.surveyId)}`;
             } else {
-                window.history.back();
+                document.location.reload();
```

Comment:
In the commit message,

> "while it should close the session and get back to the survey
form."

This should be updated. We do not expect that, we expect the thank you don't we? And you're allowed to make a shorter message altogether :)

Reply:
Actually, the tour was expecting that the close button returns to the survey back-end form but it is better the way we change it. I have not changed that part (as I have seen your message after updating commit) but I have explained why we have changed the tour and why it is better that way (little sense to show the survey back-end form on a screen that is meant to be displayed in public).